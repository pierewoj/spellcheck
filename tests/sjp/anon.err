3,171 AAAA: aaa aara
19,153 itp: atp dtp ftp ibp ifp ijp imp ip isp it ita iti itr itt itu itw iwp tp vtp
20,219 tampona: tampon tamponad tamponu tampony tarpona
23,110 codzień: codzie odzień
28,297 itp: atp dtp ftp ibp ifp ijp imp ip isp it ita iti itr itt itu itw iwp tp vtp
36,253 wienia: bienia cienia gienia hienia mienia pienia wiania widnia wiednia wiena wienie wiesia wiśnia wlenia wpienia wżenia zienia
