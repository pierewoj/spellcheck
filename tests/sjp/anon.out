---
Kilka tygodni temu pojechałem na weekend z moją dziewczyną do jej rodziców. Gdy w obudziłem się rano w sobotę pierwszą moją myślą było, jak u większości facetów “Trzeba iść się odlać”. Byłem niewyspany i półprzytomny, ale ciśnienie na pęcherz zmusiło mnie do niezwłocznego udania się do łazienki.
Otworzyłem drzwi od łazienki a tam… mama mojej dziewczyny, zupełnie goła, z wielkimi wiszącymi cycami. Byłem w takim szoku że krzyknąłem pierwsze co mi przyszło na myśl “#AAAA MOJE OCZY!” i zatrzasnąłem drzwi, słysząc z drugiej strony jej śmiech. Chciałem uciec do pokoju, ale w panice chyba straciłem orientację przestrzenną bo przywaliłem głową w futrynę drzwi, tak że aż mi pociemniało przed oczami.
Dobrze, że “teściowa” jest wyluzowana i nie wspominała już o tej sytuacji.
---
Byłam na wakacjach w małym miasteczku. Szłam chodnikiem i minął mnie nieduży pies z foliowym workiem w pysku. Po chwili przeszedł przez ulicę na przejściu dla pieszych. Zaintrygował mnie.
Byłam ciekawa po co mu ten worek, więc poszłam za nim. Śledziłam go tak może 200 metrów, gdy nagle wszedł do sklepu. Sprzedawca wziął worek z pyska psa, wyjął z niego monety i dał mu gazetę. Pies z gazetą w pysku wyszedł ze sklepu i pobiegł z powrotem tą samą drogą. Spytałam sprzedawcę co to za pies, odpowiedział, że codziennie tak przychodzi po gazetę dla swojej 85-letniej pani.
Wcześniej bałam się psów, ale od tego dnia patrzę na nie inaczej.
---
To było 10 lat temu przy mojej komunii. Gdy ciocie dowiedziały się, że czas na moją komunię, dostałam parę pięknych sukienek. Oczywiście musiałam zdecydować, którą będę miała w ten dzień, a które na "biały tydzień".
Na dzień komunii wybrałam piękną białą sukienkę z różyczką z boku, wyróżniałam się z tłumu jednakowych sukienek. Na przyjęciu gdzie była rodzina dostałam parę kopert z pieniędzmi. Uznałam że je schowam, ponieważ starszy brat mówił, że jak oddam rodzicom to one znikną. Koperty schowałam, ale niestety zapomniałam gdzie. W jakiś czas po nieudanych poszukiwaniach każdy zapomniał o tym incydencie.
Parę tygodniu temu w osiemnaste urodziny postanowiłam, że pozbędę się zabawek i rzeczy które były do oddania dla rodziny, bądź ludzi którzy ich potrzebują. Na strychu znalazłam karton, w którym była moja suknia z komunii. Wyjęłam ją i wtedy zobaczyłam, że pudło z boku jest rozwalone, a na dnie poza jakimiś duperelami są koperty.
Wyjęłam je i nie mogłam uwierzyć, znalazłam swoje komunijne pieniądze. Prawie dwa tysiące złotych leżało na dnie kartonu przez dziesięć lat. To był najlepszy prezent na moje osiemnaste urodziny, do tej pory wszyscy się śmieją że odkładałam na osiemnastkę.
---
Weszłam do McDonalda z zamiarem kupienia sobie kawy. Było dużo ludzi. Zajęłam miejsce na końcu długiej kolejki. Zwróciłam uwagę że przede mną w kolejce jest 7-8 upośledzonych dorosłych mężczyzn. Obok stała starsza kobieta, ich opiekunka, i ich nadzorowała. Gdy mnie zauważyła powiedziała “Nam tu trochę zejdzie, proszę przejść przed grupę”. Następnie głośno powiedziała do swojej grupy, żeby “przepuścić panią”. Podziękowałam i przeszłam. Idąc patrzyłam na każdego mijanego mężczyznę czy jest upośledzony czy nie, żeby się zorientować gdzie kończy się ich grupa. Gdy doszłam do pierwszego z nich, popatrzyłam na niego, a on na mnie. Uśmiechnęłam się, powiedziałam “dziękuję” i zajęłam miejsce w kolejce przed nim.
Parę sekund później poczułam stukanie w ramię. Odwróciłam się i ten właśnie mężczyzna z załamaną miną zaczął mi wyjaśniać, że on nie jest z tą grupą. Chciałam go przeprosić i powiedziałam “Najmocniej przepraszam, myślałam że Pan jest…” i w tym momencie przerwałam, zdając sobie sprawę że nie mam jak sensownie zakończyć tego zdania, nie pogarszając sytuacji.
Gość był po prostu wyjątkowo mało atrakcyjny, a ja wzięłam go za upośledzonego. Przesunęłam się, aby go przepuścić, ale zrobił tylko zmieszaną minę i zaraz pośpiesznie opuścił kolejkę i skierował się do drzwi. Wiedziałam, że pewnie zepsułam mu humor na cały dzień, czułam się strasznie.
---
W domu do pewnego czasu miałam typowy babiniec - ja , mama i młodsza siostra. Relacje między naszą trójką były bardzo kumpelskie, nie było tematów tabu #itp. O wszystkim można było "na luzie" porozmawiać. Pewnego dnia, gdy miałam okres, miałam iść na imprezę. Malowałam się w łazience, a moja młodsza siostra stała obok i się przyglądała.
Pomiędzy robieniem jednej kreski a drugiej, krzyknęłam do mamy czy są jeszcze tampony. Moją siostrę bardzo zainteresował ten 'temat' i pyta się rodzicielki jakie to uczucie, kiedy kobieta ma w sobie tego nieszczęsnego #tampona. W żartobliwej odpowiedzi usłyszała: "To tak, jakbyś uprawiała seks". Na co ja, stojąc już z tamponem w ręku, patrząc to na moją siostrę, to na tampon powiedziałam: "Mamo, jeśli według ciebie to takie samo uczucie, to nawet nie wiesz, ile w życiu Cie ominęło".
Mina rodzicielki połączona ze śmiechem siostry - bezcenne.
---
U mnie w pracy wszyscy koszmarnie przeklinają i mi się to udziela w rozmowach z kolegami z pracy, mimo że na #codzień rzadko klnę. Parę tygodni temu jechałem autobusem i do mnie zadzwonił kolega. Opowiadał mi o pewnym wydarzeniu tego dnia i w pewnym momencie powiedziałem do niego zdanie "Nie wiem, jakaś kur*a zamknęła okno".
Tak się złożyło, że kilka sekund wcześniej kobieta stojąca obok mnie w autobusie też zamknęła okno i myślała, że mówię o niej. Odwróciła się i takiego liścia mi zasadziła, że do wieczora miałem czerwony ślad na policzku. Tak mnie rozbawiła ta sytuacja, że nawet nie miałem jej za złe.
Ale od tamtego czasu postanowiłem sobie, że koniec z tym przeklinaniem, nawet z kolegami z pracy.
---
Pochodzę z małej miejscowości. Nigdy mi to nie przeszkadzało, bo i tak jestem odludkiem i nie potrzebuję towarzystwa. Gdy przeprowadziłem się na studia do dużego miasta to była to dla mnie wielka zmiana. Uznałem, że czas trochę się otworzyć na ludzi.
Ta przygoda zdarzyła mi się zaraz w pierwszym tygodniu studiów, czyli 7 lat temu. Padał deszcz, a mi jak na złość akurat uciekł autobus. Stałem na przystanku z wizją czekania 15 minut na następny, gdy zagadał do mnie starszy facet, tak ok. 50 lat. Że pogoda kiepska, że też nie zdążył na autobus #itp. Na koniec zaproponował “Może zamiast stać w deszczu skoczymy na kawę, o tu zaraz obok, do kawiarni?”. Pomyślałem, że to pewnie taki wielkomiejski zwyczaj i zgodnie ze swoim postanowieniem otwierania się na ludzi zgodziłem się.
Poszliśmy zatem do tej kawiarni, usiedliśmy przy stoliku. Kawa była tam droga, ale ku mojemu zdziwieniu facet spytał, czy może za mnie zapłacić. Co za miła niespodzianka, miasto akademickie to widać ludzie wspierają studentów. Chętnie się zgodziłem, każdy grosz się liczy.
Siedzieliśmy z tą kawą, facet opowiadał mi o sobie, jaką ma pracę, że się rozwiódł, że mieszka sam. Jednak gdy zaczął opowiadać, że ma w mieszkaniu dużą wygodną wannę, to trochę poczułem się nieswojo. No ale wytłumaczyłem to sobie, że pewnie miły starszy pan bardzo dba o higienę. Dokończyliśmy kawę i powiedziałem, że muszę już iść, a on na to, że mnie odprowadzi.
Gdy wyszliśmy z kawiarni po paru metrach chwycił mnie za ramię. W tym momencie nie miałem już wątpliwości, że coś tu jest poważnie nie tak, nawet jak na duże miasto. Szedłem jak sparaliżowany, nie wiedziałem jak tu zareagować. Wtedy facet wypalił “Mógłbym cię podotykać? Może wpadniesz na kąpiel do mnie?”. Odpowiedziałem tylko “Nie!”, co on chyba uznał za odpowiedź na tylko drugie pytanie, bo złapał mnie za tyłek! Wyrwałem mu się i jak sprinter pomknąłem na przystanek, gdzie akurat stał autobus. Wskoczyłem do środka i z ulgą zobaczyłem, jak drzwi zamykają się za mną. To że to nie był mój autobus, to był już nieistotny szczegół.
---
Mój tata łatwo się wkurza i na prima aprilis zawsze robiłem mu dość wredne kawały, żeby go bezkarnie wpienić. Tak wiem, jestem okropny. Jednak w tym roku postanowiłem dla odmiany zrobić coś miłego, ale dla równowagi poprzedzić to czymś wyjątkowo chamskim.
Tata przyszedł do domu i chciał coś zrobić na swoim laptopie. Nic mu się nie udawało, bo komputer działał bardzo powoli, co chwilę wyskakiwały dziwne okienka, reklamy. Komputer ledwo zipał. Owszem laptop był leciwy i przestarzały, totalny grat, ale ja jeszcze tego dnia wcześniej pieczołowicie mu zainstalowałem tyle dziadostwa z internetu, ile tylko dałem radę. Zawołał mnie zirytowany i powiedział, że lepiej żeby to nie był jakiś głupi żart, bo na komputerze ma bardzo ważne dokumenty.
Z głupim uśmieszkiem, bo ledwo mogłem się powstrzymać żeby się nie śmiać, powiedziałem że zaraz na to popatrzę, ale najpierw podłączę go kablem do sieci, bo może to coś z wifi. Podniosłem laptopa i idziemy z nim na przedpokój. Jak tylko byłem nad płytkami to udałem, że się potknąłem i laptop poleciał na ziemię z głośnym hukiem. Tata aż jęknął. Chwycił się za głowę i klęknął, a ja wtedy jeszcze najpierw jedną nogą nadepnąłem, a potem udając że nie mogę złapać równowagi jeszcze kopnąłem tego laptopa, mówiąc przy tym “Ups, ale ze mnie niezdara”. Ojciec czerwony, miał mord w oczach jak na mnie patrzył. Śmiejąc się uciekłem do swojego pokoju, on biegł za mną krzycząc.
Gdy tylko wpadł do pokoju wręczyłem mu nowego laptopa, którego dla niego kupiłem. Wyjaśniłem, że wszystkie jego dane i ustawienia przeniosłem, a sprzęt jest gotowy do pracy. Jeszcze nigdy nie widziałem aby ktoś tak szybko przeszedł z ekstremalnego wku*#wienia do zupełnego uszczęśliwienia.
---
FIN
